tests/test_projectbuilder.py

```diff
@@ -28,19 +30,13 @@
     PermissionError = OSError
 
 
-DUMMY_METADATA = '''
+DUMMY_METADATA = """
 Version: 1.0.0
 Provides-Extra: some_extra
-'''.strip()
+""".strip()
 
 
-DEFAULT_BACKEND = {
-    'build-backend': 'setuptools.build_meta:__legacy__',
-    'requires': [
-        'setuptools >= 40.8.0',
-        'wheel'
-    ]
-}
+DEFAULT_BACKEND = {'build-backend': 'setuptools.build_meta:__legacy__', 'requires': ['setuptools >= 40.8.0', 'wheel']}
```

Comment:
This is the one place where having it in separate lines is better, just add the trailing within the array and dictionary ``,`` and black will format it on separate lines.

Reply:
I agree, and I dislike black doing this. Is there a workaround so that black doesn't do this?

Reply:
I had the `,` before but didn't work. Let me try again.

Reply:
Ah, it does work, sorry!